### DISCUSSION TITLE
Seaweed Metrics for 2 clusters on same prometheus

### DISCUSSION THREAD

**Discussion by Alexconquer** [seaweedfs/seaweedfs#4912] [Q&A]
Hello,

I'm using metrics with victoriametrics for my first cluster.
but I don't know how to add a second cluster and be able to differentiate them.

can you help me ?
Thanks

**Comment by bogue1979** [comment]
This can be solved on prometheus side when you configure 2 different jobs for each cluster.
One of the labels will be the configured job name.

**Reply by Alexconquer** [reply]
Thanks, but I forgot to mention that : I use prometheus for Seaweed via the Push Gateway.
